packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -682,6 +686,7 @@ const DateTimePicker = ({
           className={classnames(`${iotPrefix}--date-time-picker__menu`, {
             [`${iotPrefix}--date-time-picker__menu-expanded`]: isExpanded,
           })}
+          style={{ ...updatedStyle }}
```

Comment:
for next time- this spread is unnecessary. Could just pass in the variable you created. 
```suggestion
          style={updatedStyle}
```